integrated_channels/api/v1/blackboard/views.py

```diff
@@ -1,39 +1,17 @@
 """
-    Viewset for integrated_channels/v1/blackboard/
+Viewsets for Blackboard
```

Comment:
Can we put this part of the comment back? It's the easiest way to find it when debugging: 
integrated_channels/v1/blackboard/